pymc/sampling/forward.py

```diff
@@ -89,7 +89,10 @@ def get_vars_in_point_list(trace, model):
         names_in_trace = list(trace[0])
     else:
         names_in_trace = trace.varnames
-    vars_in_trace = [model[v] for v in names_in_trace if v in model]
+    traceable_varnames = {
+        var.name for var in (model.free_RVs + model.observed_RVs + model.deterministics)
```

Comment:
Why should we include `observed_RVs` here? Will they ever take their values from the trace at this point?

Reply:
Yeah I think you're right, they won't ever be in the trace either.

Reply:
Fixed